src/vanilla/utils/devtools.ts

```diff
@@ -10,14 +80,10 @@ type Message = {
 
 const DEVTOOLS = Symbol()
 
-type EnhancerOptions = Parameters<
-  NonNullable<(typeof window)['__REDUX_DEVTOOLS_EXTENSION__']>['connect']
->[0]
```

Comment:
Though, this looks better. Curious how this is failing?

Reply:
Oh, I see. We still need to install the package.
I wonder if we can type check on it.

Reply:
Curious why L124 doesn't cause a type error.

Reply:
because it disappears, okay.